src/views/Profile/Profile.jsx

```diff
@@ -0,0 +1,25 @@
+import { useProfile } from '../../context/profilesContext';
+
+export default function Profile() {
+  const { profile } = useProfile();
+
+  return (
+    <>
+      {profile.name ? (
```

Comment:
Instead of a ternary here, since you only want to have something happen  in one case, it might be better to use something like `{ !profile.name && <>...</>}`